shopwise/src/reducer/productReducer.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+
+const productReducer = (state,action) => {
+    // if(action.type==="SET_LOADING"){
```

Comment:
Remove the commented code from PR, if you need later you can refer previous commits.